### PR TITLE
Use icons for keptn menu instead of text

### PR DESCRIPTION
**Pull request by laneli**
Improves #3643 
Use icons instead of labels for the menu.
When hovering over it the DT Overlay shows the name of the page to navigate to.

image

Signed-off-by: Elisabeth Lang <elisabeth.lang@dynatrace.com>

### CODE REVIEW

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Comment by johannes-b** [comment]
The implementation looks good, but let's wait for feedback on the icons before merging it.

**Comment by laneli** [comment]
@johannes-b I already talked with Mikele they were suggestions by him.

**Pull request merged**
